doc/articles/features/uno-material.md

```diff
@@ -6,4 +6,59 @@ Uno Material is an add-on package which lets you apply [Material Design styling]
 - Styles for existing WinUI controls like Buttons, TextBox, etc.
 - Custom Controls for Material Components not offered out of the box by WinUI, such as Card and BottomNavigationBar.
 
-For complete instructions on using Uno Material in your projects, including a set of Sketch files for designers, [consult the readme](https://github.com/unoplatform/Uno.Themes/blob/master/README.md).
\ No newline at end of file
+For complete instructions on using Uno Material in your projects, including a set of Sketch files for designers, [consult the readme](https://github.com/unoplatform/Uno.Themes/blob/master/README.md).
+
+## Getting Started
+See: [How to use Uno.Material](../guides/uno-material-walkthrough.md)
```

Comment:
I would put a stripped-down version of the set-up here (add NuGet packages, call Uno.Material.Resources.Init(), override Color resources if desired), and then link to the how-to for more detail